inspirehep/config.py

```diff
@@ -433,7 +433,8 @@ def _(x):
         list_route='/literature/db',
         item_route='/literature/<pidpath(literature):pid_value>/db',
         default_media_type='application/json',
-        search_factory_imp='inspirehep.modules.search.query:inspire_search_factory'
+        search_factory_imp='inspirehep.modules.search.query:inspire_search_factory',
+        update_permission_factory_imp="invenio_records_rest.utils:allow_all"
```

Comment:
Update permission needs to be added to https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/records/permissions.py#L89 allowing superadmin and catalogers to update records.